.github/pull_request_template.md

```diff
@@ -1,21 +1,37 @@
 ---
 name: Pull Request
-about: A simple pull request template.
-issue: Issue # if applicable
+about: Request to pull code changes into the repository.
+title: ''
+assignees: ''
 
 ---
 
-Provide a simple description of what the PR achieves. 
+## Relevant Issues
+<!-- List relevant issue tickets here. -->
+<!-- Say "Closes #0" for issues that the PR resolves, replacing 0 with the issue number. -->
 
-## Pull Request Details
 
-Please ensure your PR fulfills the following criteria:
+## Description
```

Comment:
To be clear, I think this should be a description of the implementation rather than the features, assuming the PR has a relevant issue.

Reply:
That's the intention, yep. Hence why it's saying "Describe the changes" in the comment under this. Is there something that should be changed to make this more clear?

Reply:
I think "implementation" instead of "changes" would be clearer.